arm_controllers:

  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 400  # in Hz

  joint0_effort_controller:
    type: effort_controllers/JointEffortController
    joint: waist_yaw_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}
  joint1_effort_controller:
    type: effort_controllers/JointEffortController
    joint: shoulder_pitch_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}

  joint2_effort_controller:
    type: effort_controllers/JointEffortController
    joint: shoulder_roll_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}

  joint3_effort_controller:
    type: effort_controllers/JointEffortController
    joint: shoulder_yaw_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}

  joint4_effort_controller:
    type: effort_controllers/JointEffortController
    joint: elbow_pitch_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}

  joint5_effort_controller:
    type: effort_controllers/JointEffortController
    joint: elbow_yaw_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}

  joint6_effort_controller:
    type: effort_controllers/JointEffortController
    joint: wrist_pitch_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}

  joint7_effort_controller:
    type: effort_controllers/JointEffortController
    joint: wrist_roll_l_joint
    pid: {p:  1.0, i: 0.01, d: 0.01}


 